programs/axelar-solana-governance/src/processor/gmp/schedule_time_lock_proposal.rs

```diff
@@ -12,20 +12,40 @@ use crate::events::GovernanceEvent;
 use crate::state::operator;
 use crate::state::proposal::ExecutableProposal;
 
+account_array_structs! {
+    // Struct whose attributes are of type `AccountInfo`
+    ScheduleTimeLockProposalInfo,
+    // Struct whose attributes are of type `AccountMeta`
+    ScheduleTimeLockProposalMeta,
+    // Attributes
+    // Mandatory for every GMP instruction in the Governance program.
+    system_account,
+    // Mandatory for every GMP instruction in the Governance program.
+    #[allow(dead_code)]
+    root_pda,
+    payer,
+    proposal_pda
+}
+
 /// Processes a Governance GMP `ScheduleTimeLockProposal` command.
 ///
 /// # Errors
 ///
 /// This function will return a [`ProgramError`] if any of the subcmds fail.
-pub(crate) fn process(
+pub(crate) fn process<'a>(
     ctx: ProcessGMPContext,
-    accounts: &[AccountInfo<'_>],
+    accounts: &'a [AccountInfo<'a>],
 ) -> Result<(), ProgramError> {
     let accounts_iter = &mut accounts.iter();
-    let system_account = next_account_info(accounts_iter)?;
-    let payer = next_account_info(accounts_iter)?;
-    let _root_pda = next_account_info(accounts_iter)?;
-    let proposal_pda = next_account_info(accounts_iter)?;
+
+    #[allow(clippy::unneeded_field_pattern)]
```

Comment:
The suggestion to avoid this is to use `..`. I don't like this because if the struct changes I would rather have this statement fail to compile and me be forced to check if more changes are needed here.

Reply:
I agree :+1: